OloEngine/src/OloEngine/Networking/Replication/ComponentInterpolationRegistry.cpp

```
@@ -0,0 +1,251 @@
+            .Smooth = nullptr, // discrete state — nothing to ease during reconciliation
+        });
+
+        OLO_CORE_TRACE("[ComponentInterpolationRegistry] Registered {} default interpolatable components", s_Entries.size());
+    }
+
+    void ComponentInterpolationRegistry::RegisterDefaults()
+    {
+        EnsureInitialized();
+    }
+
+    void ComponentInterpolationRegistry::Register(InterpolationEntry entry)
```

Cppcheck (2.21.0)

[style] The function 'Register' is never used.

(unusedFunction)